css/style.css

```diff
@@ -105,6 +102,36 @@ header {
     left: 50%;
 }
 
+.about-wrapper {
+    width: 100%;
+    background-color: #DAA49A;
+    box-shadow: 5px 10px 18px #888888;
+}
+
+.about-wrapper .scroll-icon {
+    color: #AA6373;
+    position: absolute;
+    bottom: -590px;
+    left: 50%;
+}
+
+.section-main {
+    width: 100%;
```

Comment:
Might not be necessary - Turning off in inspector didn't seem to make any difference. It looks as though it's set to display: block so maybe that's doing 100% width for you already.